js/coinfalcon.js

```diff
@@ -396,7 +406,8 @@ module.exports = class coinfalcon extends Exchange {
         // TODO: test status=all if it works for closed orders too
         const response = await this.privateGetUserOrders (this.extend (request, params));
         const data = this.safeValue (response, 'data', []);
-        return this.parseOrders (data, market, since, limit);
+        const orders = this.parseOrders (data, market, since, limit);
+        return this.filterOrdersByStatus (orders, 'open');
```

Comment:
The problem with this type of filtering is that it can break the pagination, since it's throwing away some counted items... Maybe we should just rename this to `fetchOrders` if this method can return orders having other statuses. We tend to avoid "emulating the server-side functionality", if the exchange does not have a direct endpoint for it.

Reply:
@ndubel plz let me know your thoughts.